src/components/Atoms/Input/story.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Component from "./index";
+
+export default {
+  component: Component,
+  title: "Input",
+  parameters: {
+    backgrounds: {
+      default: "dark",
+    },
+  },
+};
+
+const Template = (args) => <Component {...args} />;
+
+export const Input = Template.bind({});
```

Comment:
```suggestion
export const Input = Template.bind({});
Input.args = {
  onEditComplete: (value) => console.log(value);
  defaultValue: "defaultValue";
}
```
![image](https://user-images.githubusercontent.com/83479175/219535898-84b8f510-4f04-40e0-9ea4-aabfedf918e6.png)
このエラーが出る原因は、コンポーネントがPropsとして受け取るはずの `onEditComplete`  が、storybook上でPropsとして渡せていないことです。
Storybookで表示するコンポーネントにPropsを渡すには、上記のように `.args` に対してPropsを設定します。

Reply:
対応しました

6f913bbdd2e940226b044913bd46ca4232dc062d